addons/mail/models/mail_thread.py

```diff
@@ -2479,10 +2479,13 @@ def _process_attachments_for_template_post(self, mail_template):
     # MESSAGE POST API / WRAPPERS
     # ------------------------------------------------------------
 
-    def message_mail_with_source(self, source_ref, render_values=None,
-                                 message_type='notification',
-                                 auto_commit=False,
-                                 **kwargs):
+    def message_mail_with_source(
+        self, source_ref, *,
+        render_values=None,
+        message_type='notification',
+        auto_commit=False,
+        **kwargs,
+    ):
```

Comment:
Well we have horizontal space on our 789" ultra-wide screens
```suggestion
    def message_mail_with_source(
        self, source_ref, *,
        render_values=None, message_type='notification', auto_commit=False,
        **kwargs,
    ):
```

Reply:
I will change it.

Know that from a readability standpoint for humans, 66 letters per line is the optimum (The Line-Length-Readability Rule :) ) - for programming, I find it long after nearly 100 and hate it after ~120.

Reply:
Reading left to right is easier than top to bottom, don't understand why "hardcore" developers love constraining themselves writing tiny lines. SM domina syndrom probably.

Reply:
- If too many parameters (code smell), then it shows.
- Easier to add/remove parameters when reading a diff.
- Consistency. Avoids discussion on how long the line should be.
- When typing, much easier to read.

Reply:
I don't have r+. So either you do it, or someone from my team will do it next week.